packages/react/src/PricingOptions/PricingOptions.examples.stories.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react'
+import {StoryFn, Meta} from '@storybook/react'
+import {INITIAL_VIEWPORTS} from '@storybook/addon-viewport'
+
+import {PricingOptions} from '.'
+import {Box, Grid, InlineLink} from '..'
+import {SparkleFillIcon} from '@primer/octicons-react'
+
+const decorators = Story => (
+  <Box backgroundColor="default" paddingBlockStart="spacious" paddingBlockEnd="spacious" style={{minHeight: '100vh'}}>
+    <Grid>
+      <Grid.Column span={12}>
+        <Story />
+      </Grid.Column>
+    </Grid>
+  </Box>
+)
+
+export default {
+  title: 'Components/PricingOptions/Examples',
+  component: PricingOptions,
+  decorators: [decorators],
+  parameters: {
+    viewport: {
+      viewports: INITIAL_VIEWPORTS,
+    },
+  },
+} as Meta<typeof PricingOptions>
+
+export const Example1: StoryFn<typeof PricingOptions> = () => {
```

Comment:
Nit..Could we update the name (or displayName) for these so it doesn't read as Example1 and Example2 in the sidebar please?
![Screenshot 2025-05-09 at 13 16 43](https://github.com/user-attachments/assets/99db1883-c2d7-4ebc-adc8-17b6f2644cf3)

maybe something like

```suggestion
export const CopilotIndividualPlans: StoryFn<typeof PricingOptions> = () => {
```